packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -326,7 +326,7 @@ function FrameElement({ id }) {
         <Wrapper
           ref={combinedFocusGroupRef}
           data-element-id={id}
-          {...box}
+          {...boxWithBorder}
```

Comment:
Right-clicking on the border does not include the bordered element in the select dropdown:
<img width="977" alt="Screen Shot 2022-07-28 at 16 06 16" src="https://user-images.githubusercontent.com/637548/181538296-ba358627-3cd3-4e22-bf70-7477b4bcd555.png">

Reply:
Fixed now within `useElementPolygon`.